lib/fluent/plugin/out_logdna.rb

```diff
@@ -14,11 +14,16 @@ class LogDNAOutput < Fluent::BufferedOutput
     config_param :app, :string, default: nil
     config_param :file, :string, default: nil
     config_param :ingester_domain, :string, default: 'https://logs.logdna.com'
-    config_param :request_timeout_seconds, :integer, default: 30
+    config_param :request_timeout, :string, default: '30'
 
     def configure(conf)
       super
       @host = conf['hostname']
+
+      timeout_match = @request_timeout.match(/[0-9]*/)[0]
+      timeout = timeout_match == "" ? 30 : timeout_match.to_i
```

Comment:
We should use `to_f` and `30.0` here otherwise `timeout` will be an integer and millisecond prefixes won't work

Reply:
Fixed with suggestion below